influxdb3/src/commands/serve.rs

```diff
@@ -208,6 +208,16 @@ pub struct Config {
     )]
     pub wal_max_write_buffer_size: usize,
 
+    /// Number of wal files to keep behind, wal flush does not clear the wal files immediately
+    /// instead they are only deleted when num wal files count exceed this keep behind size
+    #[clap(
+        long = "num-wal-files-to-keep",
```

Comment:
should be renamed to `snapshotted-wal-files-to-keep`. It's not the actual total number of WAL files, it's the number that have been snapshotted. So if you have 200 files that haven't been snapshotted and this setting is 50, you'd have 250 total files at that point.